pdir/configuration.py

```diff
@@ -36,6 +38,10 @@ def __init__(self):
         self._configparser = ConfigParser()
         self._load()
 
+    @property
+    def colorful_output(self):
```

Comment:
I'd name this `enable_colorful_output` so it's more clear what this does, so is the configuration key's name